code/datums/station_traits/negative_traits.dm

```diff
@@ -1,7 +1,7 @@
 /datum/station_trait/carp_infestation
 	name = "Carp infestation"
 	trait_type = STATION_TRAIT_NEGATIVE
-	weight = 5
+	weight = 3
```

Comment:
Okay yeah if this actually spawns a fuckton of carps like the event does then that is probably not something we want.

Reply:
People complain about it often; people argued that a weight of 2 is better but 3 seems like a temporary fix.